charts/application/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: application
+home: https://github.com/MediaMarktSaturn/helm-charts
+description: Generic application chart with common requirements of a typical workload.
+type: application
+maintainers:
+  - name: MediaMarktSaturn
+    url: https://github.com/MediaMarktSaturn
+appVersion: 1.0.0
+version: 2.2.1
```

Comment:
should this version be adjusted to `1.0.0`?
or the charts existed already in an artifactory?

Reply:
if the url to this chart will change from user's perspective, then I would start with `1.0.0`.
And we can't see a 'changelog' anyway when we keep this version, right?

Reply:
right. valid point. will reset it.